packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -528,18 +630,27 @@ class UserSessionManager implements IUserSessionManager {
     state.session.sessionInfo.value = {};
   }
 
+  removeItem(sessionKey: UserSessionKeysType) {
+    const entries = state.storage.entries?.value;
+    const { storage, key } = entries[sessionKey];
+    if (this.isStorageTypeValidForStoringData(storage)) {
+      const store = this.storeManager?.getStore(storageClientDataStoreNameMap[storage]);
+      store?.remove(key);
+    }
+  }
+
   /**
    * Clear storage
    * @param resetAnonymousId
    */
   clearUserSessionStorage(resetAnonymousId?: boolean) {
```

Comment:
Looks like this function is not used. So, let's remove it. Also, the `removeItem` method.